components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.hpp

```diff
@@ -64,6 +66,10 @@ class MAMMicrophysics final : public MAMGenericInterface {
   // Output extra mam4xx diagnostics.
   bool extra_mam4_aero_microphys_diags_ = false;
 
+  // number of species involved in gas phase chemistry (gases + aerosols)
+  static constexpr int num_gas_aerosol_constituents_ = mam_coupling::gas_pcnst();
+  // number of species with external forcing
+  static constexpr int extcnt_ = mam4::gas_chemistry::extcnt;
```

Comment:
I am not sure why we need `extcnt`. The diagnostic arrays are dimensioned (col,gas_pcnt, lev). Am I missing something?